README.md

```diff
@@ -1 +1,5 @@
-# hello-microverse
\ No newline at end of file
+# hello-microverse
+This project aims to showcase Github flow and Linters.
+# built with
+HTML
+CSS
```

Comment:
- Add in your ``.gitignore`` file your ``test.md`` file and ``node_modules`` directory.